rpc/server.go

```diff
@@ -137,14 +139,35 @@ func (s *Server) Listen() error {
 // listener.
 func (s *Server) Serve(handler http.Handler) {
 	s.handler = handler
+	s.activeConns = make(map[net.Conn]struct{})
+
+	server := &http.Server{
+		Handler: s,
+		ConnState: func(conn net.Conn, state http.ConnState) {
+			s.mu.Lock()
+			defer s.mu.Unlock()
+
+			switch state {
+			case http.StateNew:
+				s.activeConns[conn] = struct{}{}
+			case http.StateHijacked, http.StateClosed:
```

Comment:
I don't think so - if a connection is hijacked it means it's been handed off to another arbiter to handle (I think).